jquery.dynatable.js

```diff
@@ -857,6 +857,7 @@
 
       cache = { dynatable: { dataset: settings.dataset } };
       if (!firstPush) { cache.dynatable.scrollTop = $(window).scrollTop(); }
+      $.extend(cache, window.history.state);
```

Comment:
I was initially worried that this might introduce a bug, e.g. I had previously queried by dynatable table with a search string, and then I deleted the search string and hit enter, which now adds a new state to the pushstate history that has no query, whereas the previous state had a query; I was worried that calling extend could potentially merge the old state with the new for dynatable, but because we're only doing a shallow copy with `$.extend` above and because all of the dynatable settings are namespaced under the `dynatable` key in the cache object, the new empty dynatable settings object would overwrite the old populated dynatable object.

In short, I think we're good to go here.